vmm_tests/vmm_tests/tests/tests/multiarch.rs

```diff
@@ -445,6 +453,35 @@ async fn vmbus_relay_heavy(config: Box<dyn PetriVmConfig>) -> anyhow::Result<()>
     Ok(())
 }
 
+/// Test for SCSI to SCSI relay with TDX
+/// This test validates that storage relayed through VTL2 (OpenHCL) works for TDX VMs
+#[vmm_test(
+    hyperv_openhcl_uefi_x64[tdx](vhd(windows_datacenter_core_2025_x64))
+)]
+#[cfg_attr(not(windows), expect(dead_code))]
+async fn scsi_to_scsi_relay_tdx(config: Box<dyn PetriVmConfig>) -> anyhow::Result<()> {
+    let mut vm = config.with_vmbus_redirect(true).run_without_agent().await?;
+    vm.wait_for_successful_boot_event().await?;
+    vm.send_enlightened_shutdown(ShutdownKind::Shutdown).await?;
+    assert_eq!(vm.wait_for_teardown().await?, HaltReason::PowerOff);
+    Ok(())
+}
```

Comment:
This function body looks identical to the function body of `vmbus_relay`, so just add another entry to the list in `vmbus_relay`